src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/Helpers/macOS/SnackbarViews/ActionMessageSnackBarView.macos.cs

```diff
@@ -10,35 +10,11 @@ public ActionMessageSnackBarView(NativeSnackBar snackBar)
 		{
 		}
 
-		// Gets the maximum width of the action button. Possible values 0 to 1.
-		protected virtual nfloat ActionButtonMaxWidth => 1f;
-
 		protected override void Initialize()
 		{
 			base.Initialize();
-			foreach (var action in SnackBar.Actions)
+			foreach (var actionButton in SnackBar.Actions)
 			{
-				var actionButton = new NSButton
-				{
-					Title = action.ActionButtonText,
-					WantsLayer = true,
-					LineBreakMode = action.Appearance.LineBreakMode,
-				};
-				if (SnackBar.Appearance.Background != NativeSnackButtonAppearance.DefaultColor)
-				{
-					actionButton.Layer.BackgroundColor = action.Appearance.Background.CGColor;
-				}
-
-				if (SnackBar.Appearance.Font != NativeSnackButtonAppearance.DefaultFont)
-				{
-					actionButton.Font = action.Appearance.Font;
-				}
-
-				actionButton.Activated += async (s, e) =>
-				{
-					await action.Action();
-					Dismiss();
-				};
```

Comment:
Just a question: why did we drop it?

Reply:
it is moved to NativeSnackButton. foreach type is changed

Reply:
src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/Helpers/NativeSnackButton.ios.macos.cs line 39